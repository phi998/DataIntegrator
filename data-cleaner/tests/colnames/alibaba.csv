Product Name,Description,Description,Product Category,Seller,Age,Minimum Order,other,Product Description,Price,Description
