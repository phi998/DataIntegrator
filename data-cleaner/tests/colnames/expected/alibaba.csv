Product Name,.,.,Product Category,Seller,.,Minimum Order,.,Product Description,Price,Minimum Order
